### PR TITLE
Use much faster JDK utility for converting an int to a byte sequence.

### PR DESCRIPTION
**Pull request by CodingFabian**
At the moment Lettuce uses a private helper method accepting a long parameter to write the length of a String or byte array to redis.
First off, this length will be always "int",
secondly, the method uses a hand written StringBuilder and number % 10 algorithm to translate the long into a string.
It will produce a String 4321 for a length of 1234. This string will be reversed and the char values appended to the buffer.

The JDK however comes with the handy util Integer.toString(intValue) which will produce a correctly ordered String. This method is much faster and also benefits from improvements done to the JDK
(this was actually done quite recently: http://mail.openjdk.java.net/pipermail/core-libs-dev/2015-November/036765.html)

In summary this improves CPU and memory usage of this method.

### CODE REVIEW

**Comment by mp911de** [comment]
Merged. Thank you.
